pubspec.yaml

```diff
@@ -1,5 +1,13 @@
 name: stack_trace
-version: 1.9.4-dev
+# Note! This version number is referenced directly in the pub source code in
```

Comment:
seems this comment was deleted on master before, you may need to rebase to avoid introducing this again?

Reply:
did it.